onyo/lib/commands.py

```diff
@@ -1070,40 +1070,43 @@ def onyo_set(inventory: Inventory,
 
 @raise_on_inventory_state
 def onyo_tree(inventory: Inventory,
-              dirs: list[tuple[str, Path]]) -> None:
+              paths: list[tuple[str, Path]],
+              directories: bool = False) -> None:
     r"""Print the directory tree of paths.
```

Comment:
I'm fine with this change, but I just want to comment that I originally chose the var `dirs` because files are not a valid target for `onyo_tree()`. It must be a dir.